docs/hardware/quest-pro.mdx

```diff
@@ -254,11 +253,55 @@ In case there is a beta you would like to try, you can switch to the beta follow
 
 </details>
 
+### SteamLink Setup
+
+<details>
+  <summary>SteamLink Setup</summary>
+
+:::danger
+Do **NOT** change the OSC port settings in VRCFaceTracking for SteamLink!
+Changing either of them to 9015 **WILL CAUSE PROBLEMS AND NOTHING WILL WORK**. 
+Those VRCFT port settings are for communication with *VRChat*.
+::: 
+
+0. Follow Valve's official [Setup instructions for SteamLink](https://help.steampowered.com/en/faqs/view/0E2C-406B-9135-38A4) 
+1. [**MAKE SURE YOUR HEADSET HAS EYE AND FACE TRACKING ENABLED**](#enabling-eye-and-face-tracking-on-the-headset)
+2. Once connected to SteamVR, open the SteamVR menu, open VR Settings, and enable "Advanced Settings" by moving the toggle to "Show" in the bottom left.
+  <div style={{
+    width: '75%',
+    height: 'auto',
+    margin: 'auto',
+    display: 'block'
+  }}>
+    <img src={require("./img/quest/SVRL-VRSettings.jpg").default} alt="Navigating to the VR Settings tab" />
+  </div> 
+3. Navigate to the Setting tab "Steam Link", and enable the following options:
+   - Enable OSC
+   - Share eye tracking data to other apps on this PC via OSC
+   - Share face tracking data to other apps on this PC via OSC
+4. Set the OSC output port to ```9015 (CUSTOM: VRCFT, ETC.)```
+  <div style={{
+    width: '75%',
+    height: 'auto',
+    margin: 'auto',
+    display: 'block'
+  }}>
+    <img src={require("./img/quest/steamlinksettings.png").default} alt="Required settings for Face Tracking" />
+  </div>
+
+:::danger
+Do **NOT** change the OSC port settings in VRCFaceTracking for SteamLink!
+Changing either of them to 9015 **WILL CAUSE PROBLEMS AND NOTHING WILL WORK**. 
+Those VRCFT port settings are for communication with *VRChat*. 
+:::
+
+</details>
+
 ---
 
 ## Modules
 
-There are a total of **5** different VRCFT modules that can be used for the Quest Pro.
+There are a total of **6** different VRCFT modules that can be used for the Quest Pro.
```

Comment:
above sections says 5, this one says 6 (presumably because of the extra SteamLink module)

Reply:
5 connection methods, 6 modules (due to ALXR having 2 different modules)